### ISSUE TITLE
[AS2] Stage.displayState="fullScreen" doesn’t work

### ISSUE THREAD

**Issue by ousia** [ruffle-rs/ruffle#1562]
Neither desktop app (in Linux), nor web react to `Stage.displayState="fullScreen"`.

Used in https://www.mejorar-presentaciones.tk/mejorar-presentaciones-presentation.swf.

Relevant AS code:

``` js
function toogle_fullscreen() {
    if(Stage.displayState == "normal"){
        Stage.displayState="fullScreen";
    } else {
        Stage.displayState="normal";
    }
}

key_listen=new Object();

key_listen.onKeyDown = function() {
        case 70:
            toogle_fullscreen();
        break;
    }
};

Key.addListener(key_listen);
```

Many thanks for your help.

**Comment by ousia** [comment]
Maybe this is a clearer example.

If I play https://free-culture.tk/lfc-loader.swf with https://ruffle.rs/demo/, after pressing the button (the whole presentation), I should get the following (unrelated from the fact that loading isn’t performed [#1561]):

``` js
.on_release:
    fullscreen_button._visible = 0;
    fs_area._visible = 0;
    Stage["displayState"] = "fullScreen";
    _root.load_text = "[Click to start]";
    _root.dlprogress_mvclip._visible = true;
.end
```

_[Click to start]_ is displayed, but content doesn’t go fullscreen.

But now I realize that `Stage.displayState` needs to be implemented first (according to #274).

Now I realize that one thing is to add fullscreen capabilities (such as @relrelb did with #1515) and quite another one is to be able to deal with fullscreen statements in AS2 code.

Many thanks for your help.

**Comment by ousia** [comment]
PR #5137 has the issue.

Many thanks to @hatal175 for the code and to @kmeisthax for the merge.
